paywall/src/utils/durations.js

```diff
@@ -109,3 +111,11 @@ export function expirationAsDate(timestamp) {
 
   return MONTH_NAMES[month] + ' ' + day + ', ' + year
 }
+
+export function expirationAsText(timestamp) {
+  const text = expirationAsDate(timestamp)
+  if (timestamp - new Date().getTime() / 1000 < 86400) {
```

Comment:
What is the scenario? Is this to handle keys with less than a day prior to expiry remaining? Would you mind adding a comment here or extracting this to a method with a meaningful name?

Reply:
extracting what?

Reply:
`timestamp - new Date().getTime() / 1000 < 86400`

Reply:
thanks